sql/wsrep_schema.cc

```diff
@@ -87,6 +87,7 @@ static const std::string create_frag_table_str=
   "seqno BIGINT, "
   "flags INT NOT NULL, "
   "frag LONGBLOB NOT NULL, "
+  "xid VARCHAR(512) NOT NULL,"
```

Comment:
Hmm, what if you do migrate from 10.4 ES -> 10.5 ES ? In that case we already have this table but not this column, right ?

Reply:
Fixed in cb30963